bin/user_int.rb

```diff
@@ -0,0 +1,92 @@
+load('../lib/logic.rb')
+class UserInteface
+  attr_accessor :player1, :player2, :grid
+  @@add = {
+    1 => "11",
+    2 => "13",
+    3 => "15",
+    4 => "31",
+    5 => "33",
+    6 => "35",
+    7 => "51",
+    8 => "53",
+    9 => "55"
+  }
+
+  def initialize()
```

Comment:
In your Game class it appeared you were generating the array display also (see my comments there).